TombEngine/Objects/TR5/Entity/tr5_lion.cpp

```diff
@@ -118,94 +142,98 @@ namespace TEN::Entities::Creatures::TR5
 						break;
 					}
 
-					if (AI.ahead)
+					if (targetAhead && AI.bite)
 					{
-						if (item->TouchBits.Test(LionAttackJoints))
-						{
-							item->Animation.TargetState = LION_STATE_BITE_ATTACK;
-							break;
-						}
-
-						if (AI.distance < LION_POUNCE_ATTACK_RANGE)
+						if (AI.distance >= LION_ATTACK_MIN_RANGE)
 						{
-							item->Animation.TargetState = LION_STATE_POUNCE_ATTACK;
-							break;
+							if (AI.distance <= LION_BITE_ATTACK_RANGE)
+							{
+								item->Animation.TargetState = LION_STATE_BITE_ATTACK;
+								break;
+							}
+							if (AI.distance < LION_POUNCE_ATTACK_RANGE)
```

Comment:
Then add else if here

Reply:
Yeah, what I was answering in the previous comment. Let's keep the conversation up there.